11/umbraco-cms/extending/packages/good-practice-and-defaults.md

```diff
@@ -0,0 +1,207 @@
+---
+description: Information on good practices and common defaults for Umbraco package development.
+---
+
+# Good practice and defaults
+
+This document provides guides and notes on package development. It includes good practice guidelines that will help you maintain and support your package through multiple releases and versions of Umbraco. These good practices are not prescriptive, but offer a guide as to what often works well, and not-so-well, when developing packages for Umbraco.
+
+## App Plugins
+
+To extend the Umbraco backoffice, a package can provide files such as a `package.manifest` and AngularJS views/controllers that should be stored within the `App_Plugins` folder. It's recommended to put all files in a subfolder with a unique name, preferably using the package name, like `App_Plugins\MyPackage`.
+
+Files in the `App_Plugins` folder will be publicly available on the website even though they are not in the root folder. You should not store sensitive information in the `App_Plugins` folder.
```

Comment:
The `App_Plugins` folder is in the root of the project, so this previously correctly mentioned it not being in the `wwwroot` folder (that's served publicly by default). I've reverted this change and wrapped it in backticks:

```suggestion
Files in the `App_Plugins` folder will be publicly available on the website even though they are not in the `wwwroot` folder. You should not store sensitive information in the `App_Plugins` folder.
```